client/blocks/upload-image/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UploadImage from '../';
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+
+export default class UploadImageExample extends Component {
+	state = {
+		isUploading: false,
+		uploadedImageDataUrl: null
+	};
+
+	render() {
+		const { isUploading, uploadedImageDataUrl } = this.state;
+
+		return (
+			<div className="docs__design-assets-group">
+				<h3>Default Upload Image</h3>
+				<UploadImage
+					isUploading={ isUploading }
+					onImageEditorDone={
+						( imageBlob ) => {
+							this.setState( {
+								uploadedImageDataUrl: URL.createObjectURL( imageBlob ),
+								isUploading: true,
+							} );
+						}
+					}
```

Comment:
It's just a doc block :) But yeah, I've since moved the function to a separate one. Easier to read.